tests/ink-erc20/schema.graphql

```diff
@@ -0,0 +1,28 @@
+type Block @entity @index(fields: ["number", "timestamp"]) {
+    id: ID!
+    number: Int! @index
+    timestamp: DateTime! @index
+}
+
+type Erc20EventUndefined0 @entity @index(fields: ["blockNumber", "blockTimestamp", "contract", "eventName", "from", "to"]) {
```

Comment:
is `["blockNumber", "blockTimestamp", "contract", "eventName", "from", "to"]` a composable index for the all specified fields?

Reply:
Hm, maybe it's incorrect. I'm not sure what is rigth way to do fast sort by multiple fields?

Reply:
it depends on a specific use case. i guess generic version can be:
blockNumber + contract + eventName + <event_arg>
and for this contract it will look like:
```
@Index_(["blockNumber", "contract", "eventName", "from"], {unique: false})
@Index_(["blockNumber", "contract", "eventName", "to"], {unique: false})
```
a bit verbose but i don't know how can we do it better

Reply:
sounds good